cms/djangoapps/contentstore/features/discussion-editor.py

```diff
@@ -18,7 +18,7 @@ def i_see_only_the_settings_and_values(step):
     world.verify_all_setting_entries(
         [
             ['Category', "Week 1", False],
-            ['Display Name', "Discussion", False],
+            ['Display Name', "Discussion", True],
```

Comment:
Interesting - I'm going to investigate the behavior here to make sure nothing is changed that shouldn't be. Otherwise, if it all looks good I'll merge.